### ISSUE TITLE
_mm_cmpeq_sd intrinsic is misleading

### ISSUE THREAD

**Issue by ndrewh** [Vector35/binaryninja-api#4024]

**Comment by plafosse** [comment]
I don't think this is nitpicky at all we're really just not supporting the pseudo op:
![image](https://user-images.githubusercontent.com/10556585/226769455-12842fbb-5e46-486b-b209-ee0a8d7e6455.png)
